README.md

```diff
@@ -304,6 +304,36 @@ Android:
 
     calabash-android run path_to.apk -p android features/login.feature
     
+
+## Running on Cloud
+### Appcenter (Xamarin, formerly acquired By Microsoft)
```

Comment:
Replace all `Appcenter` with `App Center`

Remove the `(Xamarin, .... )` parenthetical